### DISCUSSION TITLE
does the docker container need an NVIDIA Graphics card

### DISCUSSION THREAD

**Discussion by shott92** [carla-simulator/carla#6137] [Q&A]
I see in the CARLA documentation about building with Docker that there are the following requirements
Docker & **NVIDIA Container Toolkit:** 

My question is around this "The NVIDIA Container Toolkit is a library and toolset that exposes NVIDIA graphics devices to Linux containers." do i need an NVIDIA graphics card to use carla in a docker container or can i use and AMD GPU, and if so do i need an AMD equivilant to expose the amdgpu to linux containers?

Thanks

**Comment by manne11** [comment]
@shott92 were you able to figure it out? I'm also a bit lost on the GPU details. in some areas of the documentation, it seems carla is GPU agnostic. However in other parts(like using docker containers and requirements for nvidia drivers) it seems to have some dependency on NVIDIA GPU's. So I'm wondering if its possible to run with an AMD gpu
